cves/CVE-2020-7471.yml

```diff
@@ -266,7 +264,7 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: A lack of extensive enough consideration in crafting their input sanitization
```

Comment:
Did your research indicate why this was the case? How might this have been improved given further consideration?

Reply:
I just thought that was the nature of SQL injections, if they get through, that means that you weren't able to sanitize your input well enough..